src/sugar3/graphics/toolbox.py

```diff
@@ -15,9 +15,14 @@
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
 
-"""
-STABLE.
-"""
+'''
+A toolbox arranges a group of toolbars in a verical fashion.
+Toolbars are assigned an index and can be accessed using this index.
+Indicies are generated in the order the toolbars are added.
+
+Emits signal `current-toolbar-changed` signal when the
+current toolbar is changed. This signal takes no arguments.
```

Comment:
Lin 68:

`self.emit('current-toolbar-changed', notebook.props.page)`

That means that the first argument is the notebook.props.page.  (whatever that is)